deprecations/migration-guide-get-reservations.md

```diff
@@ -0,0 +1,312 @@
+| `OrderItems` | [Get all order items] | Use `Id` response property in `ServiceOrderIds` request parameter. |
+| `Notes` | [Get all service order notes] | Use `Id` response property in `ServiceOrderIds` request parameter. |
+| `QrCodeData` | n/a | Use `QrCodeData` response property. |
+| `Companies` | [Get all companies] | Use `AccountId` response property where `AccountType` is `"Company"`. |
+| `AccountingStates` | [Get all payments]<br>[Get all order items] | Use `AccountingStates` request parameter for both operations. |
+
+## Response properties
+
+The following table maps the response properties from the original [Reservation (ver 2017-04-12)] entity to the properties of [Reservation (ver 2023-06-06)]. Only the changed properties are listed.
+
+| Original property | New property | Notes |
+| :-- | :-- | :-- |
+| `ChannelManagerGroupNumber` | n/a | No longer supported. |
+| `ChannelManager` | n/a | Replacement will be introduced in Q2/2025. |
+| `Origin` | `Origin` | Operations-specific values moved to `CommanderOrigin` (see Commander origin). |
+| `Purpose` | `Purpose` | No longer required. |
+| `StartUtc` | `ScheduledStartUtc`<br>`ActualStartUtc` | Early check-ins changed `StartUtc` value. Check-in time and scheduled start time are now provided in separate properties. |
+| `EndUtc` | `ScheduledEndUtc`<br>`ActualEndUtc` | Check-outs changed EndUtc value. Check-out time and scheduled end time are now provided in separate properties. |
+| `RequestedCategoryId` | `RequestedResourceCategoryId` | - |
+| `CompanyId` | `PartnerCompanyId` | - |
+| `CancellationReason` | `CancellationReason` | Optional, previously incorrectly marked as required. |
+| `OwnerId` | `AccountId` | - |
+| `Options` | `Options` | Uses Service order options. |
+| `AssignedSpaceId` | `AssignedResourceId` | Previously deprecated. |
+| `AssignedSpaceLocked` | `AssignedResourceLocked` | Previously deprecated. |
+| `AdultCount` | `PersonCounts` | Previously deprecated. |
+| `ChildCount` | `PersonCounts` | Previously deprecated. |
+| `CustomerId` | `AccountId` | Previously deprecated. |
+| `CompanionIds` | n/a | Previously deprecated, replaced by [Get all companions](../operations/companies.md#get-all-companies). |
+| `ChannelManagerId` | `ChannelManagerNumber` | Previously deprecated. |
+
+## Known issues
+
+The [Add reservation] and [Update reservation] operations work with older versions of request and response properties corresponding to [Reservation (ver 2017-04-12)] entity. An updated versions of these operations will be published in H2/2025.
+
+## Feedback and troubleshooting
+
+If you are blocked in the migration for any reason (e.g. missing a certain parameter or functionality), please let us know about it in this [short survey](https://mews.typeform.com/to/izhRLSFg).
+
+If you would like to consult our Technical Partnerships team about your migration plans, you can contact them at <partnersuccess@mews.com>.
```

Comment:
note: This is correct markdown syntax for bare URL, otherwise some renderers won't pick up the email address. It also allows adding the dot to the end of sentence.